api/v1alpha1/subnet_types.go

```diff
@@ -28,10 +28,12 @@ type SubnetSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	Ipam      string   `json:"Ipam"`
-	Addresses []IpPool `json:"Addresses"`
-	Routes    []string `json:"Routes,omitempty"`
-	DNS       string   `json:"DNS,omitempty"`
+	Ipam           string   `json:"Ipam"`
+	Address        string   `json:"Address"`
+	Mask           int16    `json:"Mask"`
+	AllocationPool []IpPool `json:"AllocationPool"`
+	Routes         []string `json:"Routes,omitempty"`
+	DNS            string   `json:"DNS,omitempty"`
```

Comment:
Here we should depict the latest changes that have happened in the Subnet CR

Reply:
Will fix in the next commit.